css/amp.css

```diff
@@ -334,3 +334,13 @@ amp-analytics {
   overflow: hidden !important;
   visibility: hidden;
 }
+
+amp-user-notification {
+  position: fixed !important;
+  top: 0;
+  left: 0;
+  overflow: hidden;
+  visibility: hidden;
+  background: rgba(255, 255, 255, .7);
+  z-index: 1;
```

Comment:
Do we need to have some form of control over the maximum height with position fixed? If we don't - the user won't be able to click the "dismiss" button.